### DISCUSSION TITLE
WoWDB Website

### DISCUSSION THREAD

**Discussion by virgo77** [azerothcore/azerothcore-wotlk#9937] [General]
hey there!
as I got my home little AC Server running pretty well, I would like to know how to install a WoWDB on my webserver, like WoWHead or [EvoWoW](https://wotlk.evowow.com) or [WotLKDB](https://wotlkdb.com)?
I just want to have this kind of DB at home because I think that in years, they may vanish and I will be very sad :(
I searched here and found some mods but none of them seems to implement a WoWDB.

I tried to contact EvoWoW or WotLKDB's admins for help but without success. I never had any answer. So, is someone here can tell me if this kind of DB Website exists?

Thanks a lot

V.

**Comment by Eronox** [comment]
Maybe a future project to be added in Azerothcore repository list 😉

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/aowow

**Reply by Kitzunu** [reply]
Now I can't guarantee that it will work as it hasnt been updated since April, and we've had a lot of changes since then. But AoWow is probably what you are looking for

**Comment by virgo77** [comment]
WoW! Oo'
I did not expect about this mod, and didn't saw it... because i searched here on the 'mod' keyword. so sorry!
I think this week-end will be a test one :)
And I'll give a feedback soon
Thanks Kitzunu
